### ISSUE TITLE
Untagged List option

### ISSUE THREAD

**Issue by R-Sandor** [R-Sandor/FindFirst#136]

**Comment by Nikshinde7** [comment]
Is this still relevant? If so could you share some more info about this? I would be happy to contribute!

**Comment by R-Sandor** [comment]
@Nikshinde7 Yeah this is still problem. So right now when you import a bookmark that has no tag it is just added to the list of bookmarks returned to the frontend. 

It would nice if there was a way to say 'get me all bookmarks that aren't tagged yet', this untagged - tag, should be in the tag list on the left hand side of the application with the total count in the count badge.

I will assign it to you! :rocket: 
Thanks!

**Comment by R-Sandor** [comment]
@Nikdhinde7 is this still in progress? If not I will change it unassigned.

**Comment by zockicookie** [comment]
Hi, I would love to take a look at this. Would you assign this to me, @R-Sandor?

Is the goal here to just add another group in the left section of my screenshot?

![Image](https://github.com/user-attachments/assets/1bc9bb68-df0d-4363-9484-fb3ab5592b69)

And also, would you take a look at the sizing of each object when it is not the unit size? I think a frontend developer should look over this and size it in "tetris-style".

**Comment by R-Sandor** [comment]
@adrianf-codes Grab it :) All yours.
